OpenRobertaServer/staticResources/js/app/simulation/simulationLogic/robot.ev3.js

```diff
@@ -284,62 +284,67 @@ define(['simulation.simulation', 'interpreter.constants', 'simulation.robot', 'g
                 }
                 utterThis.pitch = pitch;
                 utterThis.rate = speed;
-                utterThis.onend = function(event) {
+                utterThis.volume = volume;
+                utterThis.onend = function (event) {
                     that.sayText.finished = true;
                 };
-                SpeechSynthesis.speak(utterThis);
+                //does not work for volume = 0 thus workaround with if statement
+                if (volume != 0) {
+                    //cancel needed so speak works in chrome
+                    SpeechSynthesis.cancel();
```

Comment:
Seems that this cancel stops all "speaks" in the simulation, also the ones from other robots when running in multiple simulation. Can we check here, if a speak is currently running?

Reply:
Checking if a speak is running didn’t work. The issue is, in chrome for some reason the speechSynthesis object gets created with the speaking flag equaling true. This causes the queue to never progress. I now moved the cancel right after creation of "var SpeechSynthesis" so when the simulation is started it gets created and immediately canceled this seems to fix it.